src/main/java/net/minecraftforge/common/DimensionManager.java

```diff
@@ -56,9 +56,22 @@
 
 public class DimensionManager
 {
+    private static class Dimension
+    {
+        private final DimensionType type;
+        private final int delayToUnload;
+        private int ticksWaited;
+        private Dimension(DimensionType type, int delayToUnload)
+        {
+            this.type = type;
+            this.delayToUnload = delayToUnload;
+            this.ticksWaited = 0;
+        }
+    }
+
     private static Hashtable<Integer, WorldServer> worlds = new Hashtable<Integer, WorldServer>();
     private static boolean hasInit = false;
-    private static Hashtable<Integer, DimensionType> dimensions = new Hashtable<Integer, DimensionType>();
+    private static Hashtable<Integer, Dimension> dimensions = new Hashtable<Integer, Dimension>();
```

Comment:
Does anyone know why `Hashtable` is used by this class?
Also, if you want to use a primitive based map, there's an `int -> Object` map provided by fastutil.